taipy/core/scenario/_scenario_repository.py

```diff
@@ -69,7 +73,9 @@ def __to_pipeline_ids(pipelines) -> List[PipelineId]:
 
     @staticmethod
     def __to_cycle(cycle_id: CycleId = None) -> Optional[Cycle]:
-        return _CycleManagerFactory._build_manager()._get(cycle_id) if cycle_id else None
+        return (
+            _CycleManagerFactory._build_manager()._get(cycle_id) if cycle_id else None
+        )
```

Comment:
```suggestion
        return _CycleManagerFactory._build_manager()._get(cycle_id) if cycle_id else None
```

Reply:
yeah, I had an issue with black on pycharm that was fixed on 88 cols. I tought I had fixed the weird formatting, will change it.